frontend/src/components/Map/MapWrapper.tsx

```diff
@@ -0,0 +1,64 @@
+import { GlobalStyles } from '@mui/material';
+import { useAtom } from 'jotai';
+import { Projection } from 'ol/proj';
+import React, { useMemo, useState } from 'react';
+
+import { baseLayerIdAtom } from '@frontend/stores/map';
+
+import { LayerDrawer } from './LayerDrawer';
+import { Map } from './Map';
+import { Zoom } from './Zoom';
+import { createWMTSLayer, getMapProjection } from './mapFunctions';
+import { mapOptions } from './mapOptions';
+
+export function MapWrapper() {
+  const [projection] = useState(() =>
+    getMapProjection(
+      mapOptions.projection.code,
+      mapOptions.projection.extent,
+      mapOptions.projection.units
+    )
+  );
+  const [baseLayerId] = useAtom(baseLayerIdAtom);
+
+  const baseMapLayers = useMemo(() => {
+    if (!baseLayerId) return [];
+
+    return mapOptions.baseMaps
+      .filter((baseMap) => baseMap.id === (baseLayerId ? baseLayerId : 'opaskartta'))
```

Comment:
[nit] I think the default base layer ID could be stored and exported as a `const` in `stores/map.ts` to avoid errors and repetition, also nullish coalescing should work here pretty well:
```suggestion
      .filter((baseMap) => baseMap.id === (baseLayerId ?? 'opaskartta'))
```